js/main.js

```diff
@@ -0,0 +1,36 @@
+const div = document.createElement('div');
+const grayBgDiv = document.createElement('div');
+const hamburgerBttn = document.querySelector('.menu-bars');
+
+hamburgerBttn.addEventListener('click', () => {
+  div.id = 'overlay';
+  div.classList.add('overlay');
+  grayBgDiv.id = 'overlay-menu-bg';
+  grayBgDiv.classList.add('overlay');
+  div.innerHTML = '<img src="./media/cancel.png" class="cancelBttn"></img>'
```

Comment:
- [ ] kindly add the HTML code inside a nav tag and this nav inside your header tag.
According to semantics and good practices, it will be better to have your hamburger menu inside a nav tag, indicating that this is a navigation element that is in your header, this way your project not only will look more professional but you will get a better SEO score for search engines 👍 .

Reply:
Done ✔

Reply:
Well addressed 👍🏼